Source/GmmLib/CMakeLists.txt

```diff
@@ -544,4 +544,7 @@ if(ARCH EQUAL 32)
 	endif()
 endif()
 
-add_subdirectory(ULT)
+if(MEDIA_RUN_TEST_SUITE)
```

Comment:
If in case someone build GmmLib separately from Source/GmmLib, this check will fail and skip ULT run, so better to modify this as 
if(NOT DEFINED RUN_TEST_SUITE OR RUN_TEST_SUITE)

Reply:
If not defined in the first place, this option was setup to ON by default. See the top level CMakelists.